cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -218,33 +249,37 @@ describe('Script Builder', () => {
         }
       }
 
+      beforeEach(() => {
+        cy.log('empty downloads directory')
+        cy.task('deleteDownloads', {dirPath: downloadsDirectory})
+      })
+
       it('will return 0 tables and 0 rows, for an empty dataset', () => {
+        cy.log('turn off composition sync')
+        cy.getByTestID('flux-sync--toggle').click()
+        cy.getByTestID('flux-sync--toggle').should('not.have.class', 'active')
         cy.log('select empty dataset')
-        selectBucket('defbuck3')
-        cy.getByTestID('flux-editor').contains(`from(bucket: "defbuck3")`, {
-          timeout: 3000,
-        })
+        cy.getByTestID('flux-editor').monacoType(`{selectall}{enter}
+            from(bucket: "defbuck3") |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
+          `)
+        cy.getByTestID('flux-editor').contains('defbuck3')
```

Comment:
New logic now requires:
* if in composition sync
* must have bucket and measurement selected.

Specifically, this logic was already supposed to be present -- but the conditional failed because it was assuming that any text present was someone writing flux (outside the composition training wheels). Which was an incorrect assumptions. So once we fixed that --> this test  case had to be updated.